.github/workflows/natsjs.yml

```diff
@@ -0,0 +1,48 @@
+name: NATS.js CI
+
+on:
+  push:
+    tags:
+      - '*'
+    branches:
+      - '*'
+  pull_request:
+    branches: [master]
+
+jobs:
+  build:
+    name: ${{ matrix.config.kind }} ${{ matrix.config.os }}
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [10.x, 14.x]
+
+    steps:
+      - name: Checkout NATS.js
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 1
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node-version }}
```

Comment:
In doc I see `matrix.node` instead of `matrix.node-version`. Maybe they both work, but using https://github.com/nektos/act, this is choking on that variable:
```
ERRO[0000] Unable to interpolate string '${{ matrix.node-version }}' - [ReferenceError: 'version' is not defined] 
```
whereas with `${{ matrix.node }}` it works (I am running with dry mode: act -n).

Reply:
Their own doc uses it. 

https://help.github.com/en/actions/language-and-framework-guides/using-nodejs-with-github-actions#starting-with-the-nodejs-workflow-template

Reply:
Like I said, maybe both work, but nektos/act did not like that, and their own doc seem indeed to have both versions, the one I saw was using matrix.node [here](https://help.github.com/en/actions/reference/workflow-syntax-for-github-actions#example-running-with-more-than-one-version-of-nodejs)